streams/ex1.saved

```diff
@@ -98529,4 +98529,4 @@ Ordering: 0
 0.015023882
 0.0068553618
 keys
-mAXXXXXXXXXXXXXXXXXXZi66666666666666666666666666666666666666666666666i
+OOOmAXXXXXXXXXXXXXXXXXXZi66666666666666666666666666666666666666666666666i
```

Comment:
I am confused by `OOO` here, does this do anything?

Reply:
Yes, it reduces refinement. Unfortunately, `O` in 2D means switching function of `o`, but in 3D it is decreasing the order directly. This is not confusing at all to the users 🙃

Reply:
Oh, right 🤦